Samples/SubtitleGenerator/MainViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace SubtitleGenerator
+{
+    public class MainViewModel : ObservableObject
+    {
+        private bool _controlsEnabled;
+        public bool ControlsEnabled
+        {
+            get => _controlsEnabled;
+            set => SetProperty(ref _controlsEnabled, value);
+        }
+    }
+}
```

Comment:
```suggestion
namespace SubtitleGenerator;

public partial class MainViewModel : ObservableObject
{
    [ObservableProperty]
    private bool _controlsEnabled;
}
```

Use MVVM Toolkit's source generator for this. (and file-scoped namespace example, so much cleaner!) 🙂